### ISSUE TITLE
SDK based projects, with SDK 2.0 restore causes a build error

### ISSUE THREAD

**Issue by nkolev92** [NuGet/Home#5449]
Net Core SDK based projects with Net Core 2 causes a build error due to difference in the assets file.
The difference is below:
Old behavior
```
  "frameworks": {
      "netstandard2.0": {
        "dependencies": {
          "NETStandard.Library": {
            "suppressParent": "All",
            "target": "Package",
            "version": "2.0.0-preview2-25401-01",
            "autoReferenced": true
          }
        },
        "imports": [
          "net461"
        ],
        "assetTargetFallback": true,
        "warn": true
      }

```

vs old behavior:

```
    "frameworks": {
      "netstandard2.0": {
        "dependencies": {
          "NETStandard.Library": {
            "suppressParent": "All",
            "target": "Package",
            "version": "2.0.0-preview2-25401-01",
            "autoReferenced": true
          }
        },
        "assetTargetFallback": [
          "net461"
        ],
        "warn": true
      }
```

**Comment by rrelyea** [comment]
This happens with 4.3.0.4192 NuGet.Tools.vsix but not the current build inserted in VS.
@nkolev92 is investigating options.

**Comment by rrelyea** [comment]
SDK code is running in Msbuild process, not in DevEnv.
It uses NuGetPackageResolver API to read the assets file.
If the fix in 4.3.0.4192 needs to be a VS only change, (no SDK/CLI insertion) and as such, cannot change the format written to the assets file.
If it was loading the DevEnv copy of the nuget dll, it knows how to read the new format...

**Comment by nkolev92** [comment]
Redundant now.
